docs/modules/rtcmem.md

```diff
@@ -0,0 +1,64 @@
+# RTC User Memory Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2015-06-25 | [DiUS](https://github.com/DiUS), [Johny Mattsson](https://github.com/jmattsson) | [Johny Mattsson](https://github.com/jmattsson) | [rtcmem.c](../../app/modules/rtcmem.c)|
```

Comment:
I reckon you get to claim maintainership of this one here :)
Also, the path listed is incorrect.

(Is it really _seven_ years since first wrote the rtcmem module? 🤯)